result_sink_test.go

```
@@ -0,0 +1,197 @@
+				Metric:   "metric",
+				Score:    float64(i),
+			})
+		}(i)
+	}
+	wg.Wait()
+
+	f, err := os.Open(p)
+	if err != nil {
+		t.Fatalf("Open: %v", err)
+	}
+	defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)